app/configs/locationColumns.ts

```diff
@@ -1,16 +1,28 @@
 import { ColDataCardDef } from "@/types/types";
 import { Location } from "@/types/locations";
-
+import { formatDollars } from "@/utils/formatters";
 export const locationColumns: {
   [name: string]: ColDataCardDef<Location>;
 } = {
   location_id: {
-    name: "location_id",
+    path: "location_id",
     label: "Location ID",
   },
-  // todo: need to support non-lookup table relationships
-  //   cr3_crash_count: {
-  //     name: "cr3_crash_count",
-  //     relationshipName: "locations_list_view"
-  //   }
+  cr3_crash_count: {
+    path: "locations_list_view.cr3_crash_count",
+    label: "CR3 crashes (5 years)",
+  },
```

Comment:
again, here you can see the path magic in action, so we can now render this nested value on the location details card. not how the `relationship` property is absent—we don't need it because this field is not editable.